{
  "uid" : "cb3a19533c885825",
  "name" : "Post db create",
  "fullName" : "test_first.TestPOST#test_post_db_create",
  "historyId" : "1aec0f14b932a152b799443dcd6b951a",
  "time" : {
    "start" : 1700405905977,
    "stop" : 1700405906331,
    "duration" : 354
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: Incorrect status code",
  "statusTrace" : "self = <test_first.TestPOST object at 0x7f6cacdc1e70>\n\n    @allure.sub_suite('POST')\n    @allure.title('Post db create')\n    def test_post_db_create(self):\n        print('\\n\\nMethod POST: db_create')\n        result_post_db_list = API.post_db_create(sid)\n>       Checking.check_status_code(result_post_db_list, 201)\n\ntest_first.py:81: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresult = <Response [500]>, status_code = 201\n\n    @staticmethod\n    def check_status_code(result: requests.models.Response, status_code: int):\n        \"\"\"\n        Method check status code\n        :param result: Response JSON\n        :param status_code:\n        :return: int\n        \"\"\"\n>       assert status_code == result.status_code, 'Incorrect status code'\nE       AssertionError: Incorrect status code\n\nutils/checking.py:17: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: Incorrect status code",
    "statusTrace" : "self = <test_first.TestPOST object at 0x7f6cacdc1e70>\n\n    @allure.sub_suite('POST')\n    @allure.title('Post db create')\n    def test_post_db_create(self):\n        print('\\n\\nMethod POST: db_create')\n        result_post_db_list = API.post_db_create(sid)\n>       Checking.check_status_code(result_post_db_list, 201)\n\ntest_first.py:81: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresult = <Response [500]>, status_code = 201\n\n    @staticmethod\n    def check_status_code(result: requests.models.Response, status_code: int):\n        \"\"\"\n        Method check status code\n        :param result: Response JSON\n        :param status_code:\n        :return: int\n        \"\"\"\n>       assert status_code == result.status_code, 'Incorrect status code'\nE       AssertionError: Incorrect status code\n\nutils/checking.py:17: AssertionError",
    "steps" : [ {
      "name" : "Method POST",
      "time" : {
        "start" : 1700405905977,
        "stop" : 1700405906331,
        "duration" : 354
      },
      "status" : "passed",
      "steps" : [ {
        "name" : "Add request",
        "time" : {
          "start" : 1700405905977,
          "stop" : 1700405905977,
          "duration" : 0
        },
        "status" : "passed",
        "steps" : [ ],
        "attachments" : [ ],
        "parameters" : [ {
          "name" : "url",
          "value" : "'https://dbend.areso.pro/db_create'"
        }, {
          "name" : "method",
          "value" : "'POST'"
        } ],
        "attachmentsCount" : 0,
        "shouldDisplayMessage" : false,
        "hasContent" : true,
        "stepsCount" : 0
      }, {
        "name" : "Add response",
        "time" : {
          "start" : 1700405906331,
          "stop" : 1700405906331,
          "duration" : 0
        },
        "status" : "passed",
        "steps" : [ ],
        "attachments" : [ ],
        "parameters" : [ {
          "name" : "result",
          "value" : "<Response [500]>"
        } ],
        "attachmentsCount" : 0,
        "shouldDisplayMessage" : false,
        "hasContent" : true,
        "stepsCount" : 0
      } ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "url",
        "value" : "'https://dbend.areso.pro/db_create'"
      }, {
        "name" : "cookie",
        "value" : "{'sid': 'c599084b-bfea-42d9-8566-2c74221a6f3a'}"
      } ],
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : true,
      "stepsCount" : 2
    }, {
      "name" : "Body: <!doctype html>\n<html lang=en>\n<title>500 Internal Server Error</title>\n<h1>Internal Server Error</h1>\n<p>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.</p>\n",
      "time" : {
        "start" : 1700405906331,
        "stop" : 1700405906331,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false,
      "stepsCount" : 0
    } ],
    "attachments" : [ {
      "uid" : "b47713106166e062",
      "name" : "stdout",
      "source" : "b47713106166e062.txt",
      "type" : "text/plain",
      "size" : 336
    } ],
    "parameters" : [ ],
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "hasContent" : true,
    "stepsCount" : 4
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "epic",
    "value" : "POST REQUESTS"
  }, {
    "name" : "subSuite",
    "value" : "POST"
  }, {
    "name" : "suite",
    "value" : "REQUESTS POST"
  }, {
    "name" : "host",
    "value" : "fv-az888-640"
  }, {
    "name" : "thread",
    "value" : "1904-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test_first"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 1,
        "broken" : 0,
        "skipped" : 2,
        "passed" : 0,
        "unknown" : 0,
        "total" : 3
      },
      "items" : [ {
        "uid" : "ed7895604f3ad20e",
        "status" : "skipped",
        "statusDetails" : "XFAIL\n\nAssertionError: Incorrect status code",
        "time" : {
          "start" : 1700405275867,
          "stop" : 1700405276407,
          "duration" : 540
        }
      }, {
        "uid" : "442ae901971fa25",
        "status" : "skipped",
        "statusDetails" : "XFAIL\n\nFileNotFoundError: [Errno 2] No such file or directory: 'logs/log_2023-11-19_14-46-06.log'",
        "time" : {
          "start" : 1700405167541,
          "stop" : 1700405167542,
          "duration" : 1
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "cb3a19533c885825.json",
  "parameterValues" : [ ]
}